.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Run tests
+
+on: 
+  pull_request:
+    branches:
+      - develop
+      - master
+
+jobs:
+  build:
+
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6'
+    
+    - name: Install xcpretty
+      run: gem install xcpretty
+    
+    - name: Update submodules
+      run: git submodule sync --recursive && git submodule update --recursive --init
+      
+    - name: Setup simulator
+      id: version
+      run: |
+        CURRENT_SIMULATOR_UUID=$(xcrun simctl create TestDevice com.apple.CoreSimulator.SimDeviceType.iPhone-11 com.apple.CoreSimulator.SimRuntime.iOS-13-3)
```

Comment:
I suppose at some point will have to update the simulator reference. Do you see a way of automating that too? Not a blocker

Reply:
Not right now. Theoretically you could write a script to check the installed simulators and pick one

Reply:
But I think we should consider that an iteration :)

Reply:
@jmignac Good question. FYI the changes here in `build.yml` are just a port of how we did it in `travis.yml`. Another infrastructure factor to consider too is that we are running this test on a "prefab environment" `macOS-latest` which is currently catalina 10.15 and may change without our control.    

And yes I agree, this is not a blocker.